graphics/graphviz/Portfile

```diff
@@ -30,17 +22,23 @@ long_description                Graphviz is ${description}. Graph visualization
                                 simple text language, and produce output in a \
                                 variety of visual and text formats.
 
-checksums                       rmd160  8fc103fc519e07e9db522ec2f748838571c71acf \
-                                sha256  ca5218fade0204d59947126c38439f432853543b0818d9d728c589dfe7f3a421
+use_parallel_build              yes
+use_bzip2                       no
+use_xz                          yes
+
+master_sites                    ${gitlab.homepage}/-/package_files/${package_id}/download?dummy=
+
+checksums           rmd160  405efb8376de8b74c1c881587c0ad7f5756e0940 \
+                    sha256  4e048eb2b0d6eb0d9fd3236ef976fb637067452e5e802678f3c7d06115877231 \
+                    size    23558836
```

Comment:
This may seem nit-picky, but we should use consistent indentation vis-a-vis the rest of the portfile.